plaso/engine/artifact_filters.py

```diff
@@ -43,6 +43,9 @@ def __init__(self, artifacts_registry, artifact_filters, knowledge_base):
     self._artifacts = artifact_filters
     self._artifacts_registry = artifacts_registry
     self._knowledge_base = knowledge_base
+    self.find_specs_per_source_type = {
```

Comment:
If this attribute is not used outside the class (except by tests) please follow the convention to add a leading underscore:
`self._find_specs_per_source_type`

Otherwise change the class docstring and add this as an "Attribute"